src/ui/Roster.ts

```diff
@@ -312,7 +312,8 @@ export default class Roster {
          this.addMenuEntry({
             id: 'online-help',
             handler(ev) {
-               window.location = onlineHelpUrl;
+               let win = window.open(onlineHelpUrl, '_blank');
```

Comment:
This is likely to be blocked by popup blockers and noreferrer/noopener should be set. Maybe it's easier to make a fake click on the a label or something like that.

Reply:
fake click with shift ? so that it will open in a new window/tab?

Reply:
what about this:

let newTab = window.open();
newTab.location.href = onlineHelpUrl;
newTab.focus();